### PR TITLE
Add Krishnamurty Number to Math algorithms 

### PR DESCRIPTION
**Pull request by Nanak360**
I have added the Krishnamurthy Number checker algorithm. This is a common algorithm that is pretty famous here in India especially from a beginner's perspective. 
Talking of Krishnamurthy number,  it's a number whose sum total of the factorials of each digit is equal to the number itself.

**hacktoberfest-accepted**

### CODE REVIEW

**Comment by ericklarac** [comment]
Could you add some tests under the test folder?

**Comment by Nanak360** [comment]
Yes, I'll surely do that @ericklarac.
I would also like to add that I am new to opensource, motivated by Hacktoberfest. Would you please help me to make this PR count as one of my Hacktoberfest PRs? 
If you'll just add a "hacktoberfest-accepted" label to this Issue, it'll mean a lot to me.

File: algorithms/maths/krishnamurthy_number.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+def factorial(n):
+    fact = 1
+    while n != 0:
+        fact *= n
+        n -= 1
+    return fact
+
+
+def checkKrishnamurthy(n):
```
Comment (ericklarac) [inline algorithms/maths/krishnamurthy_number.py:27]: Use snake case

File: algorithms/maths/krishnamurthy_number.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    sumOfDigits = 0   # will hold sum of FACTORIAL of digits
```
Comment (ericklarac) [inline algorithms/maths/krishnamurthy_number.py:28]: Also, se snake case for the variables

**Review by ericklarac — COMMENTED** [review]
Add some test and add your implementation to the readme, you can use [this merged PR](https://github.com/keon/algorithms/pull/721/files) as a guide

**Comment by Nanak360** [comment]
For some reason, the tests failed.
I'm not understanding why. Would someone help me out?

**Comment by ericklarac** [comment]
> For some reason, the tests failed.
> I'm not understanding why. Would someone help me out?

Traceback (most recent call last):
  File "/home/travis/build/keon/algorithms/tests/test_maths.py", line 329, in test_factorial
    self.assertEqual(637816310, factorial(34521, 10**9 + 7))
TypeError: factorial() takes 1 positional argument but 2 were given.

The factorial function was previously defined in other file. You are exporting your function, and it is overriding the other function.

A workaround could be to rename the `factorial` function on `krishnamurthy_number.py`, or try only to export `krishnamurthy_number` on the `algorithms/maths/__init__.py` file.

**Comment by Nanak360** [comment]
Okay, I am doing the changes.
Could you please a "hacktoberfest-accepted" label to this PR?

**Comment by Nanak360** [comment]
Yaaaeee...!!!!
It's finally merged! <3

**Pull request merged**